src/styles/icons.css

```diff
@@ -24,51 +24,2397 @@
   -moz-osx-font-smoothing: grayscale;
 }
 
-.icon-exit:before {
-  content: "\e905";
+.icon-Exit-Door:before {
```

Comment:
lowercase all of these. css  classes are better kept case insensitive

Reply:
ah okay, this was just because the haven icon file names were capitalised so its picked up on that. Ill change this.

Reply:
Ah, I see.